users/utils.py

```diff
@@ -0,0 +1,28 @@
+import jwt
+
+from functools              import wraps
+from django.http            import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+
+from users.models           import User
+from my_settings            import SECRET_KEY, ALGORITHM
+
+def LogInDecorator(func):
+    @wraps(func)
+    def wrapper(self, request, *args, **kwargs):
+        if not request.headers.get('Authorization'):
+            return JsonResponse({'message' : 'NO_TOKEN'}, status = 403)
+        
+        try:
+            access_token = request.headers.get('Authorization', None)
+            decoded_token = jwt.decode(access_token, SECRET_KEY, ALGORITHM)
+            request.user = User.objects.get(id = decoded_token['user_id'])
```

Comment:
```suggestion
        try:
            if not request.headers.get('Authorization'):
                return JsonResponse({'message' : 'NO_TOKEN'}, status = 403)
                
            access_token  = request.headers.get('Authorization', None)
            decoded_token = jwt.decode(access_token, SECRET_KEY, ALGORITHM)
            request.user  = User.objects.get(id = decoded_token['user_id'])
```
1. 처음 if 문도 try이 안에 있는 것이 조금 더 가독성이 좋아 보이네요.
2. = 기준으로 정렬 해주세요.